engine-agent/src/main/java/io/camunda/zeebe/process/test/engine/agent/EngineControlImpl.java

```diff
@@ -96,7 +100,7 @@ public void getRecords(
             RecordResponse.newBuilder().setRecordJson(recordJson).build();
         responseObserver.onNext(response);
       } catch (JsonProcessingException e) {
-        e.printStackTrace();
+        LOG.error("Failed mapping record to JSON", e);
```

Comment:
👍 Thanks this looks better

🔧 I'm not sure what information will be available in JsonProcessingException. It might be useful to add some information about the specific record that couldn't be mapped. E.g. `LOG.error("Failed to map record {} at position {} on partition {} to JSON", record.getKey(), record.getPosition(), record.getPartitionId(), e)`